Back End Challenge/pom.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
If you could choose any other build tool besides Maven for Java, what would you choose and why?